src/components/RegistrationForm/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import Text from '@components/Text'
+import Wrapper from '@components/Wrapper'
+import Button from '@components/Button'
+import Link from '@components/Link'
+import InputWithLabel from '@components/InputWithLabel'
+import { Box, DoubleInputsWrapper, InlineWrapper } from './styles'
+
+const RegistrationForm = () => {
+  return (
+    <Wrapper mTop="regular">
+      <Box>
+        <form action="">
+          <Wrapper mTop="regular">
+            <DoubleInputsWrapper>
+              <InputWithLabel
+                id="firstName"
+                label="First Name"
+                placeholder="John"
+              />
+              <InputWithLabel
+                id="lastName"
+                label="Last Name"
+                placeholder="Doe"
+              />
+            </DoubleInputsWrapper>
+          </Wrapper>
+          <Wrapper mTop="regular">
+            <DoubleInputsWrapper>
+              <InputWithLabel
+                id="job"
+                label="Job"
+                placeholder="Product Designer"
```

Comment:
idk what’s on the mockup but should we maybe change that to “Product owner” or “Developer”? So that it’s more representative of our actual users?